apis/facebook/api/src/facebook_api.e

```diff
@@ -356,12 +426,13 @@ feature {NONE} -- Implementation
 
 	add_parameters (a_method: STRING; request:OAUTH_REQUEST; a_params: detachable STRING_TABLE [STRING])
 		do
-			if a_method.is_case_insensitive_equal_general ("GET") or else a_method.is_case_insensitive_equal_general ("DELETE") then
-				add_query_string (request, a_params)
-			else
-				check is_post: a_method.is_case_insensitive_equal_general ("POST") end
-				add_body_parameters (request, a_params)
-			end
+			add_query_string (request, a_params)
```

Comment:
missing comment that would be good to understand ...
remove commented lines.

Reply:
done